projects/helper/staking.js

```diff
@@ -60,10 +60,14 @@ function stakingUnknownPricedLP(stakingContract, stakingToken, chain, lpContract
         let token, stakedBal;
         if(token0.toLowerCase() === stakingToken.toLowerCase()){
             token = token1;
-            stakedBal = BigNumber(bal.output).times(reserveAmounts[1]).div(reserveAmounts[0]).toFixed(0);
+            stakedBal = reserveAmounts[0] !== '0'
+                ? BigNumber(bal.output).times(reserveAmounts[1]).div(reserveAmounts[0]).toFixed(0)
+                : BigNumber(0);
```

Comment:
there is a tiny bug here: it should return a string not a BigNumber object, so it must be '0'

Tiny nitpick here is, we should not be using this function with an empty LP, or if it is an empty LP, this should return tvl in quote token instead of 0 baseTokens